### ISSUE TITLE
Unnecessary command in documentation?

### ISSUE THREAD

**Issue by pfebrer** [albgar/aiida_siesta_plugin#125]
The documentation on installation here: https://docs.siesta-project.org/projects/aiida-siesta/en/latest/installation.html states that you need to run `reentry scan -r aiida` once the plugin is installed.

However, there is no mention about it in the main aiida documentation for installing plugins: https://aiida.readthedocs.io/projects/aiida-core/en/latest/howto/plugins_install.html?highlight=plugins and I checked that it works without it. Is it possible that this command is no longer needed?

**Comment by bosonie** [comment]
You are totally right, since aiida 2.0, the `reentry scan` is not necessary anymore. I will modify the documentation.
